Customer_Application/Customer_Api/Controllers/CustomerDataController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Customer_Api.Infrastructure;
+using Customer_Api.Model;
+using Dapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data.SqlClient;
+using UserData;
+
+namespace Customer_Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomerDataController : ControllerBase
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IDataBaseOperation _DataBaseOperation;
+        private readonly ILogger<CustomerDataController> _logger;
+        private readonly IMessageSession session;
+
+
+        public CustomerDataController(IConfiguration configuration, IDataBaseOperation DataBaseOperation, ILogger<CustomerDataController> logger, IMessageSession session)
+        {
+            _configuration = configuration;
+            _DataBaseOperation = DataBaseOperation;
+            _logger = logger;
+            this.session = session;
+        }
+
+        [HttpGet ("getallcustomer")]
+        public async Task<ActionResult<List<object>>> GetAllCustomerData()
+        {
+            using var connection = new SqlConnection(_configuration.GetConnectionString("DefaultConnection"));
```

Comment:
move to infrastructure

Reply:
Connection String moved in Infrastructure